python/revng/cli/support.py

```diff
@@ -54,8 +54,12 @@ def relative(path: str) -> str:
 def try_run(
     command, options: Options, environment: Optional[Dict[str, str]] = None, do_exec=False
 ) -> Union[int, NoReturn]:
-    if is_executable(command[0]):
-        command = wrap(command, options.command_prefix)
+    if len(options.command_prefix) > 0:
+        if is_executable(command[0]):
+            command = wrap(command, options.command_prefix)
+        else:
+            sh = get_command("sh", options.search_prefixes)
+            command = wrap([sh, "-c", 'exec "$0" "$@"', *command], options.command_prefix)
```

Comment:
This was added because `revng run-trace` is a python script, so it was not possible to e.g. add gdb to it
In general, I'm not sure, but I think having scripts being first-class citizens is a good thing imho.

Reply:
Gotcha. Can you push this commit to the end of the PR, right after the tracing one. So I'll decide what to do once I'll try it out.

Reply:
Moved the commit